src/gsy_e/models/strategy/infinite_bus.py

```diff
@@ -30,7 +30,9 @@
 from gsy_e.models.strategy.commercial_producer import CommercialStrategy
 
 
+# pylint: disable=missing-class-docstring, too-many-instance-attributes, too-many-arguments
 class InfiniteBusStrategy(CommercialStrategy, BidEnabledStrategy):
+    """Implemenntationn for infinite bus to participate in GSy Exchange."""
```

Comment:
Butterfly keyboards, I know the struggle :(
```suggestion
    """Implementation for infinite bus to participate in GSy Exchange."""
```